packages/dart/sshnoports/bundles/shell/headless/root_sshnpd.sh

```diff
@@ -2,14 +2,25 @@
 # disable "var is referenced but not assigned" warning for template
 # shellcheck disable=SC2154
 
-# Uncomment the following lines to specify your own values, or modify them inline below
-# device_atsign="@example_device"
-# manager_atsign="@example_client"
-# device_name="default"
+# Configuration of sshnpd service
+# This script is a template for the sshnpd background service.
+# You can configure the service by editing the variables below.
+# This service file covers the common configuration options for sshnpd.
+# To see all available options, run `sshnpd` with no arguments.
 
-sleep 10; # allow machine to bring up network
+manager_atsign="@example_client" # MANDATORY: Manager/client address (atSign)
+device_atsign="@example_device"  # MANDATORY: Device address (atSign)
+device_name="default"            # Device name
+user="$(whoami)"                 # MANDATORY: Username
+# s="-s" # Uncomment if you wish the daemon to update authorized_keys to include public keys sent by authorized manager atSigns
+# u="-u" # Uncomment if you wish to have the daemon make various information visible to the manager atsign - e.g. username, version, etc - without the manager atSign needing to know this daemon's device name
+v="-v" # Uncomment to enable verbose logging
```

Comment:
This line is not commented

Reply:
I left verbose uncommented intentionally, I think it should be on by default

Reply:
Changed to "Comment if you wish to disable"